### PR TITLE
Add headCommitDate

### PR DESCRIPTION
**Pull request by leonardehrenfried**
This PR adds the ability to read the date of the current HEAD commit.

As the return value I've chosen an ISO formatted string of the datetime instead of returning a `java.util.Date`. This is because `Date` is gross and the reliance on an old Java version does not allow using `java.time.OffsetDateTime`.

### CODE REVIEW

**Comment by typesafehub-validator** [comment]
Hi @leonardehrenfried,

Thank you for your contribution! We really value the time you've taken to put this together.

Before we proceed with reviewing this pull request, please sign the Lightbend Contributors License Agreement:

[http://www.lightbend.com/contribute/cla](http://www.lightbend.com/contribute/cla)

**Comment by leonardehrenfried** [comment]
I've signed the CLA.

File: src/main/scala/com/typesafe/sbt/git/JGit.scala [hunk 1]
```diff
@@ -91,6 +95,17 @@ final class JGit(val repo: Repository) extends GitReadonlyInterface {
     porcelain.branchList.setListMode(ListMode.REMOTE).call.asScala.filter(_.getName.startsWith("refs/remotes")).map(_.getName.drop(13))
   }
 
+  override def headCommitDate: Option[String] = {
+    val walk = new RevWalk(repo)
+    headCommit.map({ id =>
```
Comment (devstuff) [inline src/main/scala/com/typesafe/sbt/git/JGit.scala]: Nit: don't need both brackets and braces.

**Comment by leonardehrenfried** [comment]
@devstuff I removed the parenthesis.

**Comment by leonardehrenfried** [comment]
Thanks for reviewing. I believe this is good to merge. Or is there anything left for me to do?

**Comment by dwijnand** [comment]
Nope. Thanks for reminding me.

**Pull request merged**